packages/form-js-editor/src/index.js

```diff
@@ -1,41 +1,39 @@
-import { FormEditorCore } from './core';
-import { FormEditorRenderer } from './rendering';
+import FormEditor from './FormEditor';
 
 import { schemaVersion } from '@bpmn-io/form-js-viewer';
 
+export default FormEditor;

```

Comment:
Why do we export it as default? I've read somewhere that mixed default / non-default exports are discouraged.

Proposal: Export it as a normal export, similar to `createFormEditor`.

Reply:
👍🏻

Reply:
Done: https://github.com/bpmn-io/form-js/pull/75/commits/a07009ac1984bb3bdb83d81745eacc8154440263